### PR TITLE
Argocd Apps Deploy

### PR DESCRIPTION
**Pull request by voatsap**
Add function to apply an applications from ArgoCD declarations:
https://argoproj.github.io/argo-cd/operator-manual/declarative-setup/#applications

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -382,6 +382,23 @@ function main::output_software_info {
...
+#######################################
+# Deploy ArgoCD applications via kubectl
+# Globals:
+#   None
+# Arguments:
+#   None
+# Outputs:
+#   Writes progress status
+#######################################
+function deploy_argocd_apps {
```
Comment (MaxymVlasov) [inline entrypoint.sh]: I recommend change name to
```suggestion
function argocd::deploy_apps {
```

Also, just merged PR, where functions moved to `./bin/common.sh` file

File: kubernetes/apps/raw-manifest/application.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: guestbook
+  namespace: argocd
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/argoproj/argocd-example-apps.git
+    targetRevision: HEAD
```
Comment (MaxymVlasov) [inline kubernetes/apps/raw-manifest/application.yaml]: Are example will work always? Maybe good reason to specify commit SHA or tag?

File: kubernetes/charts/wordpress/values.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    db:
+      password: bar
+    rootUser:
+      password: baz
+  ingress:
+    enabled: true
+    certManager: true
+    hostname: wordpress-chart.cluster.dev
+    annotations: {
+      kubernetes.io/ingress.class: nginx,
```
Comment (MaxymVlasov) [inline kubernetes/charts/wordpress/values.yaml]: Maybe you mean
```suggestion
      kubernetes.io/ingress.class: nginx
```
?

File: entrypoint.sh [hunk 4]
```diff
@@ -451,6 +468,9 @@ for CLUSTER_MANIFEST_FILE in $(find "$CLUSTER_CONFIG_PATH" -type f); do
             # Deploy ArgoCD via Terraform
             aws::init_argocd   "$cluster_name" "$cluster_cloud_region" "$cluster_cloud_domain"
 
+            # Deploy ArgoCD apps via kubectl
+            deploy_argocd_apps
```
Comment (MaxymVlasov) [inline entrypoint.sh]:
```suggestion
            argocd::deploy_apps
```

File: kubernetes/apps/raw-manifest/application.yaml (hunk 2 above)
Comment (voatsap) [reply kubernetes/apps/raw-manifest/application.yaml]: yes it works always. its product samples

**Pull request merged**